src/core/src/rt_info.cpp

```diff
@@ -4,17 +4,19 @@
 
 #include "openvino/core/rt_info.hpp"
 
+#include "ngraph/op/util/op_types.hpp"
```

Comment:
is it possible to use `openvino` header?
nitpick: do we actually need `#include "ngraph/variant.hpp"` here? it looks like we have replaced ngraph::Variant with ov::Any